pkgs/applications/misc/procmail/default.nix

```diff
@@ -2,22 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "procmail";
-  version = "3.22";
+  version = "3.24";
 
   src = fetchurl {
```

Comment:
why isn't this using `fetchFromGitHub`?

Reply:
No particular reason. Just an oversight I guess.